cpuinfo.go

```diff
@@ -77,7 +77,7 @@ func parseCPUInfoX86(info []byte) ([]CPUInfo, error) {
 	// find the first "processor" line
 	firstLine := firstNonEmptyLine(scanner)
 	if !strings.HasPrefix(firstLine, "processor") || !strings.Contains(firstLine, ":") {
-		return nil, errors.New("invalid cpuinfo file: " + firstLine)
+		return nil, fmt.Errorf("invalid cpuinfo file: %q", firstLine)
```

Comment:
Do you think we need quoting after a colon? (We didn't quote before, so I'm wondering if this change is deliberate.)

Reply:
It is deliberate. I was thinking we should be better about passing random kernel strings to error messages with quoting.

The logging in this library is very inconsistent, so I was only trying to do some minimal cleanup.